src/QualificationRunner.Core/RunOptions/QualificationRunOptions.cs

```diff
@@ -37,5 +37,10 @@ public class QualificationRunOptions
       public string ReportConfigurationFile => Path.Combine(OutputFolder, $"{ReportConfigurationFileName}.json");
 
       public string TempFolder => Path.Combine(OutputFolder, Constants.TEMP_FOLDER);
+
+      /// <summary>
+      ///    Should simulation be performed as part of the run?
+      /// </summary>
+      public bool Run { get; set; }
```

Comment:
@Yuri05 Well in fact we have no choices. Boolean are false if not set or true if set (-r). Default is false